Sources/SwiftProtobuf/BinaryTypeAdditions.swift

```diff
@@ -99,11 +99,9 @@ public extension Message {
             throw BinaryDecodingError.missingRequiredFields
         }
     }
-}
 
-/// Proto2 messages preserve unknown fields
-public extension Proto2Message {
-    public mutating func _protobuf_mergeSerializedBytes(from bytes: UnsafePointer<UInt8>, count: Int, extensions: ExtensionSet?) throws {
+    /// SwiftProtobuf Internal: Common support for decoding.
```

Comment:
Technically don't need the first part of this comment anymore, since outsiders won't see it.